docs/cloud/cloud-databases/cloud-db-versions.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Cloud database versions
+layout: default
+parent: Manage databases
+grand_parent: Cloud
+nav_order: 4
+---
+
+# Additional database version information
+{: .no_toc }
+
+{% include page-toc.md %}
+
+
+## Where can I find my database version?
+
+### Option 1: View in your list of databases
```

Comment:
What I'd do here is just give two methods without the first heading.

Which means:
* Remove "Where can I find my database version"
* Change Option 1 to: `## Find database version in list of databases`
* Change option 2 to `## Find database version in database summary`

That way, the headings are more meaningful in the TOC and the page, and there's less to click through